src/site/content/en/handbook/quick-start/index.md

```diff
@@ -25,9 +25,9 @@ If the piece you'd like to publish is time sensitive, make sure to submit the is
 1. The tech writer will review using the [web.dev content checklist](/handbook/content-checklist) and work with you to polish the piece.
 
 ## Publishing
-1. Once you and the tech writer think the piece is good to go, submit a pull request on GitHub with your content authored in Markdown. (Check out the "[web.dev markup](/handbook/#web.dev-markup)" section for details.)
+1. Once you and the tech writer think the piece is good to go, submit a pull request on GitHub with your content authored in Markdown. Check out the [web.dev markup](/handbook/#web.dev-markup) section for details.
 1. The web.dev team will give your Markdown a once over. If everything looks good, your pull request is merged, and your piece is published! 🎉
 
 {% Aside %}
-Googlers: if you'd like to see our process in more detail, check out the [web.dev Process Document](https://docs.google.com/document/d/1Vxgrgxtz4OeJcrYJF5lpK9bVhTcGpXOXeeL4GZCV8KY).
```

Comment:
Is this hidden to just Googlers? Or does everyone see this? Feels weird to link to an internal doc, especially if this is visible to everyone.

Reply:
It does, but this was the compromise we arrived at between (a) providing a readily accessible process overview and (b) making sure Googlers who want the in-depth version of the process can access it. But I'm OK with removing it if @robdodson and @carolrivero are.

Reply:
I'm ok removing it.

Reply:
Fixed in Google Doc